src/HCSR04.cpp

```diff
@@ -24,7 +29,9 @@ double UltraSonicDistanceSensor::measureDistanceCm() {
     digitalWrite(triggerPin, LOW);
     // Measure the length of echo signal, which is equal to the time needed for sound to go there and back.
     unsigned long durationMicroSec = pulseIn(echoPin, HIGH);
-    double distanceCm = durationMicroSec / 2.0 * 0.0343;
+
+    //Cair ≈ (331.3 + 0.606 ⋅ ϑ) m/s
+    double distanceCm = durationMicroSec / 2.0 * (0.03313 + 0.0000606 * temperature);
```

Comment:
I suggest extracting speed of sound to separate variable, just to make it more clear what it is, since now it got more complex.
```suggestion
    double speedOfSoundInCmPerMs = 0.03313 + 0.0000606 * temperature // Cair ≈ (331.3 + 0.606 ⋅ ϑ) m/s
    double distanceCm = durationMicroSec / 2.0 * (0.03313 + 0.0000606 * temperature);
```